src/console_node/httpUtils.go

```diff
@@ -87,7 +87,7 @@ func postURL(URL string, requestBody []byte, requestHeaders map[string]string) (
 		return nil, -1, err
 	}
 
-	log.Printf("postURL Response Status code: %d\n", resp.StatusCode)
+	//log.Printf("postURL Response Status code: %d\n", resp.StatusCode)
```

Comment:
Same question -- should we just remove this? Or put it behind some conditional, if you're keeping it here for future debugging?

Reply:
Ibid.